solr/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+Copyright (c) 2016 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on Solr running locally. 
+
+### 1. Set Up YCSB
+
+Clone the YCSB git repository and compile:
+
+    git clone git://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn clean package
+
+### 2. Set Up Solr
+
+There must be a running Solr instance with a core or collection (when in SolrCloud mode) pre-defined. 
+The 'conf/schema.xml' file related to the core or collection must be configured to handle the expected field names during benchmarking.
+An example schema file can be found under `src/main/conf/` and is appropriate for the default field names used by the ycsb client
+
+If running in SolrCloud mode ensure there is an external Zookeepercluster running.
+
+### 3. Run YCSB
+    
+Now you are ready to run! First, load the data:
+
+    ./bin/ycsb load solr -s -P workloads/workloada -p table=<core/collection name>
+
+Then, run the workload:
+
+    ./bin/ycsb run solr -s -P workloads/workloada -p table=<core/collection name>
+
+For further configuration see below: 
+
+### Defaults Configuration
+The default settings for the Solr node that is created is as follows:
+	
+	'table'/core/collection name = 'ycsb'		// standard ycsb property
+	'solr.cloud' mode = 'false'					// expects solr to be running in stand-alone mode
+	'solr.host.list' = 'localhost:8983'			// assumes Solr is running locally on the standard port
+	'solr.commit.within.time' = '1000'			// max time in ms to wait for a commit when in batch mode
+	'solr.batch.mode' = 'false'					// commits 1 document at a time (when set to false)
+	'zookeeper.host.list' = 'localhost:2181'	// used when in SolrCloud mode only
```

Comment:
SolrCloud installations typically chroot under /solr in zookeeper, should specify that here.